src/commands/CommandType.h

```diff
@@ -44,21 +44,21 @@ class wxString;
 class AUDACITY_DLL_API OldStyleCommandType : public AudacityCommand
 {
 private:
-   ComponentInterfaceSymbol mSymbol;
+   mutable ComponentInterfaceSymbol mSymbol;
```

Comment:
There is a misspelling in the first commit comment.

There is another use of mutable, which is not in a very important class.  Still, when you do use mutable or const_cast to cheat strict const correctness, you should either think about how to eliminate that, or write a comment to explain why it is defensible.

What is a good excuse for mutable?
* you are caching some computation, and you can prove that the cache is never invalidated by later changes of the data that the result depends on.
* the computation may be expensive.

More examination of this case tells me the first is correct but the second may not be.

Can you simply eliminate the mutable variable?  It is easy in this case.  The member variable mSymbol is used for nothing else but to impement `OldStyleCommandType::GetSymbol`.

Could you move the assignment of mSymbol to some earlier place in initialization of the `OldStyleCommandType` object?  (As I suggested for the other mutable in the other pull request.)

Actually, no, not in this case!

The only obvious place to try that is in the constructor of `OldStyleCommandType` -- but when you are still constructing a base class object, the virtual function overrides defined by the derived class (here, `BuildName` is the function) cannot be called yet.  That is a C++ language rule.

I wrote a lot about an unimportant bit of code but the principles should be remembered for other cases.

So, either comment about mutable mSymbol, or just remove it.  I say remove it.